input/docs/handbook/when-activated/index.md

```diff
@@ -41,6 +41,12 @@ public class ActivatableViewModel : ISupportsActivation
 
 # View
 
+## How to ensure `ActivatableViewModel` gets activated?
```

Comment:
Maybe worth mentioning what the "ActivatableViewModel" exactly is. 
A class that implements `ISupportsActivation` or so